### DISCUSSION TITLE
[第三陣営/コンビネーション役職] : 児雷也・綱手・大蛇丸（三竦み）

### DISCUSSION THREAD

**Discussion by UzomuZo** [yukieiji/ExtremeRoles#107] [Ideas]
### 役職名 / Role Name
児雷也・綱手・大蛇丸（三竦み、tripartite deadlock）

### 役職の色 / Role color
３職別の色なら児雷也 赤・綱手 緑・大蛇丸 青
３職同じ色なら灰色系統

### 役職の説明 / Role description
各職は自身に強い役職(天敵)と弱い役職が存在し、天敵がいない状態（キルorサイドキック）で生存して試合が終了すると勝利、天敵がいる状態で敗北となる。
（例：児雷也は綱手に強く、大蛇丸に弱いため、大蛇丸がいる状態で終了すると敗北）
そのため3人残っている場合は3人とも敗北となる。

### 役職の能力 / Role ability

1.ゲーム開始時の説明に天敵が表示される。
そのため、自身を積極的に吊ろうとする相手は自身を天敵とする役職の可能性がある。

2.身代わりの術（3．土遁の術とクールタイム共有）
能力発動中にキルされると自身のみがタイムマスターのシールドのように設定した時間巻き戻る。
巻き戻るのは自分だけなので相手はすぐにキルできない（特殊キルは除く）が、周りの人に人外だとバレる。

3．土遁の術(という名のベント)
ベント使用可。（2.身代わりの術とクールタイム共有）

### 他陣営への影響 / Effect on other teams
クルー：
メリット
・身代わり発動時に近くにいればキル役職を報告して減らせる。
・生存しやすい能力なので人数が一度に減りにくい。
デメリット
・3人の枠がクルーではなくなる。

インポスター：
メリット
・三竦みが１人確定で落ちていれば、仲間と三竦みのうち天敵がいない誰か一人でパワープレイできる。
（7人盤面で仲間3人＋三竦み1人でPP）
デメリット
・身代わりでキルが無駄になり、追放される可能性がある。

ジャッカル：
一人がサイドキックされたら天敵が消えた一人は勝利できる。

### 陣営 / Team
- [ ] :クルー/Crewmate
- [ ] :インポスター/Impostor
- [x] :ニュートラル/Neutral

**Comment by yukieiji** [comment]
本当に申し訳ないんですが、それぞれがどういう役職になっているかわからないので詳しく書いてもらってもよろしいでしょうか？

**Comment by UzomuZo** [comment]
三竦みで均衡してるので全員同じ役職能力で、ただ勝利条件がそれぞれの天敵がいない状態で生存になっているだけです。

**Comment by yukieiji** [comment]
> 三竦みで均衡してるので全員同じ役職能力で、ただ勝利条件がそれぞれの天敵がいない状態で生存になっているだけです。

インポスターやクルーがこの役職をキルするメリットはあるのでしょうか
クルーやインポスター、他陣営勝利にも何にも関係ないこの役職を排除することで生存者の数が減るというデメリットはあるけど自陣営にとってのメリットがない気がします

**Comment by UzomuZo** [comment]
> インポスターやクルーがこの役職をキルするメリットはあるのでしょうか 
確かにメリットがないので、身代わりを妨害する別の能力にするかキルボタンを付与するかになりますが、キル役職が3人増えると厳しいと思うので別の能力案を考えました。

能力案１．変わり身(＋身代わり)の術
10秒程度生存するプレイヤーの姿になり、その間にキルされると変身が解けるかわりに変わっていた姿のプレイヤーが代わりに死ぬ。
自身：生存しやすい。位置が浮きやすい。
クルー：自身が安全な位置にいてもキルされる。議論が混乱する。
インポスター：キルしても生存してキル陣営だと報告される。仲間と思って他のプレイヤーをキルしたら通報される。

能力案２．影縫いの術
自身の周辺のプレイヤーが2～5秒移動不可(その場で出来るタスク、特殊能力などは可能)
自身：追われても逃げやすい。自分だけ動けるので役職がバレる。
クルー：タスクに行くのが遅れる。キルクールが溜まってキルされやすくなる。
インポスター：キルに行くのが遅れる、逃げられる。

能力案３．分身の術
自身の10秒前の行動をする分身を作り出す。分身はキルや能力を受けても無効化して行動し続ける。
自身：目撃されやすい。
クルー：分身に能力を使うと無駄になる。
インポスター：分身をキルするとキルが無駄になる。

**Comment by melonjagaimo** [comment]
キルボタン持ちが3人増えると厳しいのは同意します。
それでは、「特定の条件下でのみ児雷也・綱手・大蛇丸がキルボタンを持つ」というのはどうでしょうか？

・3人とも生きている間は、睨み合っているのでキルボタンを持たない。

・児雷也・綱手・大蛇丸のうち誰かが死亡や役職上書きを受けた場合、天敵がいなくなったプレイヤーがキルボタンを持つ。
例えば大蛇丸が最初にキルされた場合、天敵がいなくなった児雷也がキルボタンを持つ。その後児雷也が追放されたら綱手がキルボタンを持つ。

**Comment by UzomuZo** [comment]
> キルボタン持ちが3人増えると厳しいのは同意します。 それでは、「特定の条件下でのみ児雷也・綱手・大蛇丸がキルボタンを持つ」というのはどうでしょうか？
> ・3人とも生きている間は、睨み合っているのでキルボタンを持たない。
> ・児雷也・綱手・大蛇丸のうち誰かが死亡や役職上書きを受けた場合、天敵がいなくなったプレイヤーがキルボタンを持つ。 例えば大蛇丸が最初にキルされた場合、天敵がいなくなった児雷也がキルボタンを持つ。その後児雷也が追放されたら綱手がキルボタンを持つ。

なるほど、いい案だと思います。代替案も微妙な気がしていたのでその案を採用させてください。

**Comment by yukieiji** [comment]
> 3人とも生きている間は、睨み合っているのでキルボタンを持たない。
児雷也・綱手・大蛇丸のうち誰かが死亡や役職上書きを受けた場合、天敵がいなくなったプレイヤーがキルボタンを持つ。
例えば大蛇丸が最初にキルされた場合、天敵がいなくなった児雷也がキルボタンを持つ。その後児雷也が追放されたら綱手がキルボタンを持つ。

結局、この役職をインポスターやクルーがキルするメリットがない問題が未解決な気がします
3人から減らなければほぼ安全にこの役職を処理できてしまいしますし

**Comment by UzomuZo** [comment]
うーん、難しいですね。

天敵が死んだら能力ボタンが変わるとかは可能なんでしょうか？（TORの弁護士？依頼人が死んだら役職ごと変わりますが）
例えば天敵が生きている間は前に案を出した妨害の能力、死んだら身代わりで生存用の能力になるとか。

**Comment by yukieiji** [comment]
> 天敵が死んだら能力ボタンが変わるとかは可能なんでしょうか？（TORの弁護士？依頼人が死んだら役職ごと変わりますが）
例えば天敵が生きている間は前に案を出した妨害の能力、死んだら身代わりで生存用の能力になるとか。

能力を変化することは出来ますが、出されている能力だと妨害する能力が無い気がします(議論は混乱自体はしますけどクルーやインポスターへの勝利を妨害してない感じですかね)

**Comment by UzomuZo** [comment]
各陣営の人数を減らす以外の勝利を妨害ってのがよくわからなくなってきたので案と自分の考えをメモ書きします。
　クルー勝利：タスク完了、全キル陣営を排除。
⇔　　　妨害：タスク進行遅延、キル陣営の判明 排除阻止、死体通報阻止？
　インポスター勝利：自陣営が過半数以上、自陣営以外のキル陣営を排除。
⇔　　　　　　妨害：インポスター排除、キル阻害？


案１．心変わり　他のプレイヤーに一時的に全陣営からキルできるようになる役職？属性？を付与。ただしアサシンやヒーロー、ヴィランなどの特殊なキル無効相手には効かない。
【シェリフやインポスターによる同士討ちで各陣営排除、キル陣営判明阻止】

案２．焙烙玉　設置して時間経過で爆破して範囲内のプレイヤーをキルする。
手間になりそうだが途中で可視化する？ 例：設置から5～15秒で可視化、さらに5～15秒後爆破。
【時間がかかる特殊キルによる各陣営排除、可視化した場合はタスクや移動、キル阻害】

案３．煙幕　サボタージュの停電とは別で、インポスター含む他陣営を視界を狭くする、あるいは視界が狭くなるエリアを設置。視界なのでヤンデレやバウンティなどの矢印には弱い。
【キル陣営の判明阻止、キル相手以外の位置がわからないことでキル阻害、インポスターの連携キル阻害】

案4．死体操作（口寄せ）　死体に能力を使用すると生きた姿に戻して一定時間行動(停止？左右移動？)させたのちに死体に戻る。動いている間は通報不可、再キルも可能にすればキルが無駄になる。
【キル陣営の判明阻止、キル阻害】

**Comment by yukieiji** [comment]
> 各陣営の人数を減らす以外の勝利を妨害ってのがよくわからなくなってきたので案と自分の考えをメモ書きします。

やり方は幾つかありますが、いちばん簡単なのは「理不尽な勝利を押し付ける」(アリスや妖狐とか)とかでしょうか、あとはゲーム速度を落とすような妨害、マップの変形、視界効果とかでしょうか

> 案１．心変わり
案4．死体操作（口寄せ）

これは実装的に結構面倒くさい気がします

> 案２．焙烙玉
案3

これは問題ないんじゃないでしょうか

**Comment by UzomuZo** [comment]
> やり方は幾つかありますが、いちばん簡単なのは「理不尽な勝利を押し付ける」(アリスや妖狐とか)とかでしょうか、あとはゲーム速度を落とすような妨害、マップの変形、視界効果とかでしょうか

視界効果が案3ですね。
案3の煙幕はこの役職が3人いることを考えるとエリア設置（TORのヴァンパイアのニンニク系）のほうがいいのかなと思います。
ただ、設置した場所の中心から誰が置いたかばれてしまうのはさけたいところなので遅延発動させるか範囲広めにするかはしたいです。

案2は提案されてるマイナーと少し被ってるので微妙かな。
そういえば案2も前の天敵消えたらキルつける案もキル役職になるから勝利条件が変わりますかね？
（非キルならインポ2＆児雷也＆クルー1でインポ＆児雷也勝利、あるいはクルー＆児雷也＆綱手でクルー＆児雷也勝利？）

**Comment by yukieiji** [comment]
少し確認です、UzomuZoさん的にこの役職をどんなバランスにして、それがどんな感じに動いてどのような勝利を目指しているのかを確認したいです

> 案2は提案されてるマイナーと少し被ってるので微妙かな。

案2は特殊キルボタンを3人追加させるようなものなのできつい気がします

**Comment by UzomuZo** [comment]
元々のイメージは、3人が拮抗しててそれが崩れないと勝利できないけど、他の2人を消す能力がないので吊られるなりキルされるなりで消えるまで自分が生き残って勝利する、生存重視の職でした。
そこで他陣営がキルするメリットがない問題があるっていうので妨害できる能力を考えていました。

自分の中でも有力なのは案3だったので大丈夫です。
一応案2に関してはキルできるにしてもなるべく難しいようにしたつもりですが。

**Reply by yukieiji** [reply]
> 元々のイメージは、3人が拮抗しててそれが崩れないと勝利できないけど、他の2人を消す能力がないので吊られるなりキルされるなりで消えるまで自分が生き残って勝利する、生存重視の職でした。
そこで他陣営がキルするメリットがない問題があるっていうので妨害できる能力を考えていました。

限定的な状況下だと人を消せるってのは一つありなのかなって思いました

> 自分の中でも有力なのは案3だったので大丈夫です。
一応案2に関してはキルできるにしてもなるべく難しいようにしたつもりですが。

案2もありかなって思いましたが3人もそれが出来るとゲーム速度をかなり落とすので問題かもしれないって思いました

**Comment by yukieiji** [comment]
これもちょっとコンセプト等も含めて考える必要があるためディスカッションに移動します

**Reply by UzomuZo** [reply]
ディスカッションへの移動対応ありがとうございます。

> 限定的な状況下だと人を消せるってのは一つありなのかなって思いました

限定的な状況下っていうのがよくわからないんですが、緊急タスクが発動してない＆能力を押して数秒接触した場合とか、会議で自分に投票した相手＆一定距離内に自分と対象以外に人がいる状態で能力押すとか、そういう感じでしょうか？（あくまで例でこういう条件にしたいわけではないです）

**Reply by yukieiji** [reply]
> 限定的な状況下っていうのがよくわからないんですが、緊急タスクが発動してない＆能力を押して数秒接触した場合とか、会議で自分に投票した相手＆一定距離内に自分と対象以外に人がいる状態で能力押すとか、そういう感じでしょうか？（あくまで例でこういう条件にしたいわけではないです）

まぁそんな感じです

**Comment by UzomuZo** [comment]
とりあえず考えた条件案を投稿しておきます。

案1.　天敵以外 （キルできる天敵とは？となってしまうので）
案2.　煙幕、停電による視界妨害中（不意打ち、暗殺のイメージ　エアシップの暗室も？）
相手が視界妨害を受けているという条件が難しいなら自分がその効果範囲にいるとき
案3.　会議中に一定数投票された人（人数によるが3～5票程度？もしくは累計　自分が入れた票も含む　暗殺依頼）
案4.　シェリフのようにキルに回数制限つける（人数次第で1～2回、３人が無制限にキルできないように）
有害な人外判定されるとキル残数が無いときに厳しいか？

**Reply by yukieiji** [reply]
大変、返信が遅れてしまって申し訳ないです
日を開けて再度確認しましたがやはりキル能力をもたせるのはバランス的に難しい気がします(限定的な条件もかなり厳し目？デフォルトのイーターレベル位の条件が必要だと思います)
なので、この役職に求められているのは「能力や何らかのアクションでクルーやインポスターに別の誰かを排除してもらう」ということだと思います(一つの能力としてヴィジランテのエマージェンシーコールの全体版もありといえばありだと思いますが少し弱いと思います)

**Reply by UzomuZo** [reply]
>なので、この役職に求められているのは「能力や何らかのアクションでクルーやインポスターに別の誰かを排除してもらう」ということだと思います(一つの能力としてヴィジランテのエマージェンシーコールの全体版もありといえばありだと思いますが少し弱いと思います)

排除してもらう能力を考えてみましたがなかなか難しいですね。

案1. 全体エマージェンシーとバウンティ
能力ボタンを押して矢印が出た相手(天敵)を、効果時間中(5～15秒)に誰かがキルするとキルした本人の次回キルクールが0.5～1割程度？短縮される（無理に狙わなくてもいい程度）
キルがない陣営は、キル陣営が集まるので離れるか、現場目撃もしくは通報のために近づく。

案2. 接近時暴発を付与
能力ボタンを押すと一定時間(5～10秒くらい、自身が離れるための時間)後から効果が発動し、相手(天敵)に効果時間中(3秒くらい？)に近づいたひとがキルしてしまう。
近づいた人がキルボタンが無くても1回で効果が切れる。

案3. 一部設備の対象外にする
能力ボタンを押すと、相手(天敵)がアドミンやミラHQのセンサー等の設備に映らなくなる。
位置偽装容疑で吊られやすくなる。
